riscv/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 
 - Fixed `sip::set_ssoft` and `sip::clear_ssoft` using wrong address
+- Fixed conditional compilation selectors for `riscv::register::macros`
```

Comment:
https://github.com/rust-embedded/riscv/pull/203 has not been released yet (i.e., users of the released version have never been affected by the bug), so I don't think the changelog entry for this is needed.

Reply:
Where should it go then, or are you saying it doesn't need a `CHANGELOG.md` entry at all?

Asking because the CI check fails without it an entry.

Reply:
I added a tag to skip the changelog CI

Reply:
> I added a tag to skip the changelog CI

Sounds good, I'll remove it when I rebase the changes on #205.